src/html-duration-picker.js

```diff
@@ -327,9 +327,11 @@ export default (function() {
       scrollUpBtn.setAttribute('aria-label', 'Increase duration');
       scrollUpBtn.setAttribute('style', `text-align:center; width: 16px;padding: 0px 4px; border:none; cursor:default;
         height:${(picker.offsetHeight/2)-1}px !important; position:absolute; top: 1px;`);
+      scrollUpBtn.classList.add('scroll-up');
```

Comment:
Please help me understand, what is the purpose or effect of this?

Reply:
Having the css class on the elements makes them customizable.

For instance, I needed to hide the buttons in my use case.

Reply:
I see, you're right! That's brilliant